workflow/state-machines.rst

```diff
@@ -32,7 +32,7 @@ Below is the configuration for the pull request state machine.
         framework:
             workflows:
                 pull_request:
-                    type: 'state_machine'
+                    type: 'single_state'
```

Comment:
Right -- when I use 'state_machine' as the type it fails.  Am I missing something?